__tests__/app.test.js

```diff
@@ -38,19 +40,50 @@ describe("GET /api/topics", () => {
       });
   });
 });
-describe('GET apis', () => {
-    test('returns 200 status code', () => {
-        return request(app).get("/api").expect(200)
-    })
-    test('returns an object containing api info', () => {
+describe("GET apis", () => {
+  test("returns 200 status code", () => {
+    return request(app).get("/api").expect(200);
+  });
+  test("returns an object containing api info", () => {
     return request(app)
-    .get('/api')
-    .expect(200)
-    .then(response => {
-        const data = response.body
-        expect(data.apis).toHaveProperty('GET /api/topics')
-        expect(data.apis).toHaveProperty('GET /api')
-        expect(data.apis).toHaveProperty('GET /api/articles')
-    })
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        const data = response.body;
+        const endpoints = require("../endpoints.json");
+        expect(data.apis).toEqual(endpoints);
+      });
+  });
+});
+describe("GET /api/articles/:article_id", () => {
+  test("returns 200 status code", () => {
+    return request(app).get("/api/articles/2").expect(200);
+  });
+  test("responds with correct article object with article_id property", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then((response) => {
+        const { article } = response.body;
+        expect({ article }).toEqual({
```

Comment:
Please use a more flexible jest matcher here to allow further expansion of this endpoint in the future. Something like: .toMatchObject() or .objectContaining().